static/js/sheets/SheetContent.jsx

```diff
@@ -0,0 +1,520 @@
+import Component from "react-class";
```

Comment:
When moving the logic here from `static/js/Sheet.jsx` did you copy-paste or use `git cp`? That would preserve the file's history, as opposed to showing that you wrote everything with the current commit date :-)

Reply:
I didn't know about 'git cp'.  I'm wondering if I should try reverting the commit that moved the SheetContent component into its own file and then using 'git cp'.  I'm worried though that this might cause a mess downstream in the other branches that are dependent on SheetContent.jsx.  What do you think?